.env.sample

```
@@ -1,2 +1,13 @@
 export NX_SELF_HOSTED_REMOTE_CACHE_SERVER=https://cache.nickthesick.com
-export NX_SELF_HOSTED_REMOTE_CACHE_ACCESS_TOKEN=g8@ucL8em4*Z9TKXDY9OEX@!upf^Nz9
\ No newline at end of file
+export NX_SELF_HOSTED_REMOTE_CACHE_ACCESS_TOKEN=g8@ucL8em4*Z9TKXDY9OEX@!upf^Nz9
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NX_SELF_HOSTED_REMOTE_CACHE_ACCESS_TOKEN key should go before the NX_SELF_HOSTED_REMOTE_CACHE_SERVER key

(UnorderedKey)